docs/api/wrapper/setValue.md

```diff
@@ -1,6 +1,13 @@
 ## setValue(value)
 
-Sets the value of a text `<input>`.
+Sets the value to an input element of type text.
```

Comment:
Sets the value to an input element of type text.

`Sets value of a text-control input element and updates `v-model` bound data`